packages/api/src/users/users.controller.ts

```diff
@@ -33,6 +34,14 @@ export class UsersController {
       // get channel from user
       const [channel] = await this.youtube.getChannels(user)
 
+      // Ensure channel exists
+      if (!channel) {
+        throw new YoutubeAuthorizationError(
+          ExitCodes.CHANNEL_NOT_FOUND,
+          `No Youtube Channel exists for user ${user.email}`
```

Comment:
Just noticed that `user.email` isn't available at this point. When I got the error I got in the console: 
```
No Youtube Channel exists for user undefined
```
Even though I used a real google account.

Reply:
Hmm, strange. Every google account/user always has an associated email address. Can you please try logging [user](https://github.com/zeeshanakram3/youtube-synch/blob/ac5e2d4dd0fd8778959539e116e8d96f9993deef/packages/api/src/users/users.controller.ts#L31-L32) object, and see what it contains?

Reply:
It seems that depends on how you sign in with google. We're using Atlas "redirect" mode, we don't use `GoogleOAuthProvider`, so no popup is showing up. This is what I'm getting when I redirect in Atlas:
```
User {
  id: undefined,
  email: undefined,
   ...
}
```
If I use a popup with the library like it's implemented here in `/packages/app` I'm getting a proper response(with email and id).

Reply:
Ok. Then I think I will log a generic error message without stating the email address.